Data/Input/default.xci

```diff
@@ -430,7 +430,7 @@ event=Mode default
 event=StatusMessage Pilot event announced
 event=Logger note PEV
 event=PilotEvent
-label=Pilot\nEvent
+label=Pilot Event\nAnnounce
```

Comment:
I don't think the verb should be at the end; I mean, read it out loud and tell me it's good.